tests/test_orchestration/test_orchestrator.py

```
@@ -200,6 +200,52 @@ def test_empty_list(self):
         assert _interceptor_specs([]) == []
 
 
+class TestStartProxyFinishReason:
+    def _run(self, svc):
```

Ruff (0.15.18)

[warning] Missing return type annotation for private function `_run`

(ANN202)